inference/generativeai/llm-workshop/lab8-Inferentia2-gpt4all-j/inferentia2-llm-GPT4allJ.ipynb

```diff
@@ -0,0 +1,929 @@
+{
```

Comment:
<p>If you get legal approval, we will need to store this model in out sample notebook bucket. This is because if this model gets removed from huggingface in the future, this notebook will automatically break - customers often complain about broken links and references regarding our examples. I will help you store this model and you can just simply download from there using S3.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3966/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>